rott/modexlib.c

```diff
@@ -106,7 +106,7 @@ void GraphicsMode ( void )
 	uint32_t flags = SDL_WINDOW_RESIZABLE | SDL_WINDOW_ALLOW_HIGHDPI;
 	uint32_t pixel_format;
 
-	unsigned int rmask, gmask, bmask, amask;
+	uint32_t rmask, gmask, bmask, amask;
```

Comment:
Aren't these equivalent types?

Reply:
It probably depends on the platform. GCC 14 gives the following error here:

```
note: expected ‘Uint32 *’ {aka ‘long unsigned int *’} but argument is of type ‘unsigned int *’
```

Reply:
Okay, I see.